onnxscript/tests/function_libs/torch_lib/error_reproduction.py

```diff
@@ -0,0 +1,120 @@
+from __future__ import annotations
+
+import pathlib
+import platform
+import sys
+import time
+import traceback
+from typing import Any, Mapping
+
+import numpy as np
+import onnx
+import onnxruntime as ort
+import torch
+
+_REPRODUCTION_TEMPLATE = '''\
+import google.protobuf.text_format
+import numpy as np
+from numpy import array, float16, float32, float64, int32, int64
+import onnx
+import onnxruntime as ort
+
+# Run n times
+N = 1
+
+onnx_model_text = """
+{onnx_model_text}
+"""
+
+ort_inputs = {ort_inputs}
+
+# Set up the inference session
+session_options = ort.SessionOptions()
+session_options.graph_optimization_level = ort.GraphOptimizationLevel.ORT_DISABLE_ALL
+onnx_model = onnx.ModelProto()
+google.protobuf.text_format.Parse(onnx_model_text, onnx_model)
+
+# Uncomment this line to save the model to a file for examination
+# onnx.save_model(onnx_model, "{short_test_name}.onnx")
+
+onnx.checker.check_model(onnx_model)
+session = ort.InferenceSession(onnx_model.SerializeToString(), session_options, providers=("CPUExecutionProvider",))
+
+# Run the model
+for _ in range(N):
+    ort_outputs = session.run(None, ort_inputs)
```

Comment:
Any plan to extend this script to cover mismatch repro as well?

Reply:
Yes but I think I will do it in a separate pr (i.e. for the issue reshape with 0 dim for example)